.changeset/wicked-pens-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": patch
+---
+
+fix: connectApp add getVersion to wait when the device is not unlocked [LIVE-2611]
```

Comment:
>_Improve locked device detection by sending getVersion first since it's blocking when locked_ 

or something like this? I feel it's not super clear what we are doing in that changelog